torchci/pages/api/drci/drci.ts

```diff
@@ -270,15 +271,30 @@ export function reorganizeWorkflows(
         pr_number: pr_number,
         head_sha: workflow.head_sha,
         jobs: new Map(),
+        merge_base: "",
       });
     }
     const name = workflow.name!;
-    const existing_job = workflowsByPR.get(pr_number).jobs.get(name);
+    const existing_job = workflowsByPR.get(pr_number)?.jobs.get(name);
     if (!existing_job || existing_job.id < workflow.id!) {
       // if rerun, choose the job with the larger id as that is more recent
-      workflowsByPR.get(pr_number).jobs.set(name, workflow);
+      workflowsByPR.get(pr_number)!.jobs.set(name, workflow);
     }
   }
+
+  // clean up the workflows - remove workflows that have jobs
```

Comment:
Why do you want to remove workflows that have jobs?

What's the problem that this is trying to solve?

Reply:
if you dont, you get lists like "Lint, lintrunner, toc failed", except Lint is the name of the workflow, which is usually not preferred when you could just get the actual jobs that failed (lintrunner and toc in this case)